index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  <title>My Portfolio</title>
+</head>
+<body>
+
+  <header>
+
+    <div id="settings">
+      <div id="settings-date">
+
+      </div>
+      
+      <div class="settings-fa">
+        <i class="fas fa-signal"></i>
+        <i class="fas fa-wifi"></i>
+        <i class="fas fa-battery-full"></i>
+      </div>
+    </div>
+
+    <div id="union">
+      <img src="images/Icons/Icon - Menu.png" alt="union">
+    </div>
+
+    
+  </header>
+
+  <section class="headline">
+
```

Comment:
- [ ] Nice work on the header section in general, however, there are still some points that need to be worked to match the Figma design 🔽 

## Your Design 
![2023-03-23_232317](https://user-images.githubusercontent.com/104319462/227387650-7cdb95f9-49a9-4f8d-83e7-741009f16145.png)

## Figma Design
![2023-03-23_232733](https://user-images.githubusercontent.com/104319462/227387662-f229a2d2-ffc0-445d-b70e-63ba5056bc32.png)